webapp/views/wpt-app.js

```diff
@@ -109,6 +118,16 @@ class WPTApp extends PathInfo(WPTFlags(TestRunsUIBase)) {
 
       <div class="separator"></div>
 
+      <template is="dom-if" if="[[showBSFGraph]]">
+        <info-banner>
```

Comment:
Suggestion (non-blocking): Might be worth playing with making the entire banner clickable, and remove the arrow. (And change the text accordingly).